services/vaadin-directory/vaadin-directory-rating-count.service.js

```diff
@@ -23,7 +23,7 @@ module.exports = class VaadinDirectoryRatingCount extends BaseVaadinDirectorySer
         pattern: 'rating-count/:packageName',
         namedParams: { packageName: 'vaadinvaadin-grid' },
         staticPreview: this.render({ ratingCount: 6 }),
-        keywords: ['vaadin-directory', 'rating-count', 'rating count'],
```

Comment:
I suppose this _technically_ would make this badge no longer search-discoverable via `rating count`, though that's fine IMO because that term seems to be very specific to this service, which is search-discoverable through a multitude of other terms 👍